### PR TITLE
Bash kernel on binder

### PR DESCRIPTION
**Pull request by mwouts**
Hello @takluyver ,

Thank you for the Bash Kernel, I find it so useful to share or demo bash scripts!

Would you like to provide a link to a binder with the bash_kernel? This is what this PR does.

The link in the README will work only once this PR is merged. Meanwhile you can test it at
https://mybinder.org/v2/gh/mwouts/bash_kernel/bash_kernel_on_binder

If you like we could also think of
- documenting how to do this in the README
- adding `jupytext` to `binder/requirements.txt` to allow Jupyter to open `.sh` files as Bash notebooks

Let me know what you think,

Marc

### CODE REVIEW

**Comment by kdm9** [comment]
Hi @mwouts, I've recently taken on co-maintaining `bash_kernel`, and I'm currently closing all old/stale issues and PRs.

Does this PR still work with the latest version of binder? I don't use binder myself so I have no idea. If this is still both functional and useful, I'd be happy to merge it (unless @takluyver objects).

If this isn't useful any more, let me know and I'll close it (or you can)

File: binder/postBuild [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (kdm9) [inline binder/postBuild:1]: I *think* you'd need to ensure that these files don't end up in the sdist tarballs, right?

Comment (takluyver) [reply binder/postBuild:1]: My default starting point is for sdists to have everything that's checked into git, so you could recreate a rough version of the project's history from sdists if the git repo went away. These files are tiny, so I'd just leave them in the sdist.

**Comment by mwouts** [comment]
Hi @kdm9 , yes I expect this PR to work with the latest version of `bash_kernel`.

The simplest way to test this assumption would be to rebase the branch and then click on the link https://mybinder.org/v2/gh/mwouts/bash_kernel/bash_kernel_on_binder to make sure the Bash kernel is indeed available there.

Should I do the rebase or do you want to do it yourself?

**Comment by mwouts** [comment]
By the way, congratulations @kdm9 for becoming a maintainer of this project!

**Comment by kdm9** [comment]
> Should I do the rebase or do you want to do it yourself?

If you could, please do it yourself, as then your link above will work. I assume it wouldn't if I did it as it would be in this repo.

File: binder/postBuild (hunk 1 above)
Comment (kdm9) [reply binder/postBuild:1]: That makes sense, in which case ignore my comment above.

**Comment by mwouts** [comment]
Hi @kdm9 , I've just done the merge with the main branch so this PR is up to date.

I confirm the binder link still works (https://mybinder.org/v2/gh/mwouts/bash_kernel/bash_kernel_on_binder for this PR, will be https://mybinder.org/v2/gh/takluyver/bash_kernel/master in  `README.rst`). 

Here is a screenshot of Binder with the Bash kernel available:
image

**Comment by kdm9** [comment]
Niiiice :)

Thanks a bunch @mwouts . I'm happy to merge then.

@takluyver I'll merge this later today unless I hear any objection as I think you've seen this already since you commented up thread. Or just hit the button yourself of course :)

**Comment by takluyver** [comment]
Nothing I like better than pressing buttons - thanks for getting things moving again.

I suppose we could also think about having a brief demo notebook in the repository to give people something to look at on Binder.

**Pull request merged**